### DISCUSSION TITLE
problem in google chrome with this claim

### DISCUSSION THREAD

**Discussion by 04ichim** [ruffle-rs/ruffle#2386] [General]
Hello you,
Today I bought frills for chrome connections and I get a lot of bugs and it's not a game where you can hear Flashplayer.
Maybe you can help me so that I can still play tomorrow.
Thank you very much.

![image](https://user-images.githubusercontent.com/77268642/104163625-015f9180-53f7-11eb-9d63-51e4612122cd.png)

**Comment by Poopooracoocoo** [comment]
That SWF uses ActionScript 3, the newer language for SWFs, usually from 2006 and onwards. Ruffle does not currently support it. Support for it may take several months.
